tftest.py

```diff
@@ -298,6 +299,12 @@ def __init__(self, tfdir, basedir=None, binary='terraform', env=None):
   @classmethod
   def _cleanup(cls, tfdir, filenames, deep=True):
     """Remove linked files, .terraform and/or .terragrunt-cache folder at instance deletion."""
+
+    def remove_readonly(func, path, excinfo):
+      _LOGGER.warning(f'Issue deleting file {path}, caused by {excinfo}')
+      os.chmod(path, stat.S_IWRITE)
```

Comment:
would it make sense to catch errors inside this too?

Reply:
I did think about this, but I think the purpose is only to try to fix a single issue and any other issues should be allowed to be handled upstream as they were before.